src/page-attributes.ts

```diff
@@ -36,7 +36,26 @@ function readPageAttributes() {
     throw new Error('"origin" is required.');
   }
 
-  const matches = repoRegex.exec(params.repo);
+  let repo = params.repo;
+  let repoParts = repo.split('/');
```

Comment:
"repoParts" has not changed any more, should be declared with "const"

Reply:
Agreed. Thanks for reviewing. Javascript is not what I do for living so I can only make it work.